README.md

```diff
@@ -211,6 +211,7 @@ Options:
 
 	--image/-i:		image to use for the container	default: registry.fedoraproject.org/fedora-toolbox:35
 	--name/-n:		name for the distrobox		default: fedora-toolbox-35
+	--duplicate/-d:		name of the distrobox container to use as base for a new container
```

Comment:
Should be more appropriate to add a deep description in README file about the new feature.

Reply:
Sure i'll add to this later in the file description :+1: